### PR TITLE
fix oracle tests

### PR DESCRIPTION
**Pull request by atiertant**
fix bugs in oracle tests see #1689

### CODE REVIEW

File: test/unit/schema/oracledb.js [hunk 1]
```diff
@@ -206,29 +204,27 @@ describe("Oracle SchemaBuilder", function() {
...
+    expect(tableSql[0].sql).to.equal('DECLARE PK_NAME VARCHAR(200); IS_AUTOINC NUMBER := 0; BEGIN  EXECUTE IMMEDIATE (\'ALTER TABLE "users" RENAME COLUMN "foo" TO "bar"\');  SELECT COUNT(*) INTO IS_AUTOINC from "USER_TRIGGERS" where trigger_name = \'users_autoinc_trg\';  IF (IS_AUTOINC > 0) THEN    SELECT cols.column_name INTO PK_NAME    FROM all_constraints cons, all_cons_columns cols    WHERE cons.constraint_type = \'P\'    AND cons.constraint_name = cols.constraint_name    AND cons.owner = cols.owner    AND cols.table_name = \'users\';    IF (\'bar\' = PK_NAME) THEN      EXECUTE IMMEDIATE (\'DROP TRIGGER "users_autoinc_trg"\');      EXECUTE IMMEDIATE (\'create or replace trigger "users_autoinc_trg"      BEFORE INSERT on "users" for each row        declare        checking number := 1;        begin          if (:new."bar" is null) then            while checking >= 1 loop              select "users_seq".nextval into :new."bar" from dual;              select count("bar") into checking from "users"              where "bar" = :new."bar";            end loop;          end if;        end;\');    end if;  end if;END;');
```
Comment (atiertant) [inline test/unit/schema/oracledb.js]: i know ! we already think about it but didn't find an elegant way to do it...
what do you think about it? maybe only integration tests should be done

Comment (elhigu) [inline test/unit/schema/oracledb.js]: works for me

**Comment by elhigu** [comment]
@atiertant All good to merge this?

**Comment by atiertant** [comment]
@elhigu yes, we could update test/unit/schema/oracledb.js in an other pr

**Comment by elhigu** [comment]
this is great thank you!

**Pull request merged**